python-frontend/src/main/java/org/sonar/python/semantic/v2/converter/AnyDescriptorToPythonTypeConverter.java

```diff
@@ -43,6 +47,26 @@ Descriptor.Kind.AMBIGUOUS, new AmbiguousDescriptorToPythonTypeConverter()
 
   }
 
+  public Optional<ModuleType> convertModuleType(String moduleName, String moduleFqn,
+    ModuleType parent, Map<String, Descriptor> stringDescriptorMap, boolean registerAsSubmodule) {
+    Map<String, TypeWrapper> moduleMembers = new HashMap<>();
+    for (var entry : stringDescriptorMap.entrySet()) {
+      var descriptor = entry.getValue();
+      var name = entry.getKey();
+      PythonType result;
+      String fullyQualifiedName = descriptor.fullyQualifiedName();
```

Comment:
If I understand correctly, this is the fullname from Mypy

Reply:
Yes, which represents the actual fully qualified name of the symbol (where it's defined, rather than where it's re-exposed).
Do you think it could be better named?

Reply:
It was just to be sure, I think it is strange that we didn't keep the name of the filed from Mypy